cpp/unreal_plugins/UrdfBot/Source/UrdfBot/UrdfRobotComponent.cpp

```diff
@@ -4,24 +4,21 @@
 
 #include "UrdfBot/UrdfRobotComponent.h"
 
+#include <DrawDebugHelpers.h>
+
 #include "CoreUtils/Config.h"
 #include "CoreUtils/Unreal.h"
 #include "UrdfBot/UrdfJointComponent.h"
 #include "UrdfBot/UrdfParser.h"
 
-void UUrdfRobotComponent::initializeComponent(UrdfRobotDesc* robot_desc)
-{
-    ASSERT(robot_desc);
-}
-
 void UUrdfRobotComponent::createChildComponents(UrdfRobotDesc* robot_desc)
 {
     ASSERT(robot_desc);
 
     UrdfLinkDesc* root_link_desc = robot_desc->root_link_desc_;
     ASSERT(root_link_desc);
 
-    root_link_component_ = NewObject<UUrdfLinkComponent>(this);
+    root_link_component_ = NewObject<UUrdfLinkComponent>(this, FName(robot_desc->name_.c_str()));
```

Comment:
I think I prefer not naming these. Is this necessary for something?

Reply:
I add it back for debug purpose. It makes it easier to know which component Im looking at when debug with breakpoints as there are so many components.